global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.304",
+    "version": "7.0.100",
```

Comment:
Not sure what ramifications are of this, but not worried about it now... We can try updating later if need be.

Reply:
I initially went with the highest version, but issues with the dotnet workloads knocked us down to this version.